Source/SG/SG.cs

```diff
@@ -24,17 +24,18 @@ public struct InstrumentConfiguration
             public double CarrierFrequency_Hz;
             public double AverageInputPower_dBm;
```

Comment:
Average input power on SG is a little confusing. Suggest we change to DutAverageInputPower_dBm

Reply:
makes sense to me